Woof/Woof/View/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+/// A view displaying detailed information about the sitter.
+struct DetailPetSitterView: View {
+    /// The full name of the sitter.
+    let fullName: String
+    /// The phone number of the sitter.
+    let phoneNumber: String
+
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: DetailSitterViewModel
+    var body: some View {
+        ZStack {
+            Color.App.grayLight
+                .ignoresSafeArea()
+            VStack {
+                AvatarView(url: viewModel.imageURL)
+                RatingView(rating: .constant(4))
+                ContactDelailsSectionView(image: Image.IconName.fullName, message: fullName)
+                ContactDelailsSectionView(image: Image.IconName.phone, message: phoneNumber)
```

Comment:
This data should be provided by ViewModel like this
```suggestion
                ContactDelailsSectionView(image: Image.IconName.fullName, message: viewModel.fullName)
                ContactDelailsSectionView(image: Image.IconName.phone, message: viewModel.phoneNumber)
```

Reply:
Thanks! Fixed it.